python-sdk/src/astro/table.py

```diff
@@ -132,9 +132,8 @@ def row_count(self) -> Any:
         Return the row count of table.
         """
         db = create_database(self.conn_id)
-        tb = db.get_sqla_table(table=self)
-        query = select(func.count("*")).select_from(tb)
-        return db.run_sql(query).scalar()
+        result = db.run_sql(f"select count(*) from {db.get_table_qualified_name(self)}").scalar()
```

Comment:
looking fine to me. But just wondering why the old query didn't work maybe adding error logs on the ticket would be helpful for me to understand the issue.

Reply:
concatenating a string to build a query is unsafe since it is easy to inject the wrong query though it is not that important here.
you will have to add `# skipcq: BAN-B608` to pass CI

Reply:
<img width="1982" alt="image" src="https://user-images.githubusercontent.com/92459020/199181668-531fdb66-77f4-47cd-8887-25302a6ba84e.png">
it was failing silently in Open lineage but not all the queries getting were generated for the sqla.